### ISSUE TITLE
Mobile - Cannot scroll to the right to see Sign In/ Join Scratch, need to zoom out, which makes the buttons hard to push accurately

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#580]
This happens on my iPhone 6s, Chrome, on both staging and production, but @technoboy10 did not see it on his phone (so I added the needs-repro tag so we can see what circumstances it happens in).

The issue is that when I scroll as far to the right as possible, I get to the edge of the page's content, but I don't get to the right edge of the nav bar... so to get to the right edge of the nav bar, I have to zoom out so the page is "full-screen" on my phone (meaning, I can see the whole width of the page at once), and then press Sign In (or Join Scratch). At that scale, the Sign In/ Join Scratch buttons are very tiny and hard to press accurately.

Zoom all the way out to see Sign In/ Join Scratch:
![img_7631](https://cloud.githubusercontent.com/assets/1320456/16057117/777ee438-3246-11e6-8a79-77a47cde7094.PNG)
Zoom + scroll in to the upper right corner does not show Sign In/ Join Scratch:
![img_7630](https://cloud.githubusercontent.com/assets/1320456/16057118/777fa6ac-3246-11e6-9e54-24d838a172a8.PNG)

**Comment by St19Galla** [comment]
I can confirm this happens on iPhone 5s Safari as well.

**Comment by jwzimmer** [comment]
@St19Galla Thank you! : )

**Comment by Choco31415** [comment]
I decided to test several browsers to see how they handled the nav bar. Here is what I got for mobile browsers:

iOS Safari (iPad mini): The nav bar is fixed
iOS Puffin (iPad mini): The nav bar scrolls
iOS Chrome (iPad mini): This is odd. When somewhat zoomed in, the nav bar is fixed. When zoomed in more, the nav bar scrolls slightly. In those situations, the furthest I can scroll left is the "Scratch" logo. The furthest I can scroll right is half way through the "Search" bar, meaning I can't see anything beyond that. The nav bar also scrolls slightly up & down as I pan through the page up & down.

I also decided to test desktop browsers, because this might reveal something interesting. The way I tested desktop browsers was by shrinking the window's width, then using either a trackpad or scroll bar to scroll right.

Mac Safari: The nav bar scrolls
Mac Firefox: The nav bar is fixed
Mac Chrome: The nav bar scrolls

PC IE: The nav bar is fixed
PC Firefox: The nav bar is fixed
PC Chrome: The nav bar is fixed
